src/utils/translate/en.json

```diff
@@ -116,7 +116,9 @@
   "Countries": "Countries",
   "CountriesIncluded": "Countries Included",
   "CountriesAndAreas": "Countries and Areas",
-  "CountryEstimatesExist": "Countries With Seroprevalence Estimates",
+  "CountryEstimatesExist": "Seroprevalence Estimates",
+  "CountryEstimatesNotExist": "No Seroprevalence Estimates",
+  "Not Applicable": "Not Applicable",

```

Comment:
nit: can we make this key `NotApplicable`, everything else here is pascal case

Reply:
fixed